src/main/java/uk/gov/companieshouse/web/accounts/security/WebSecurityConfigurer.java

```diff
@@ -20,27 +21,31 @@ public class WebSecurityConfigurer {
     public SecurityFilterChain govUkAccountsSecurityFilterChain(HttpSecurity http)
             throws Exception {
 
-        return http.securityMatcher("/accounts/**")
-                .addFilterBefore(new SessionHandler(), BasicAuthenticationFilter.class)
-                .addFilterBefore(new HijackFilter(), BasicAuthenticationFilter.class).build();
+        return ChsCsrfMitigationHttpSecurityBuilder.configureWebCsrfMitigations(
+                        http.securityMatcher("/accounts/**")
+                                .addFilterBefore(new SessionHandler(), BasicAuthenticationFilter.class)
+                                .addFilterBefore(new HijackFilter(), BasicAuthenticationFilter.class))
+                .build();
     }
 
     @Order(2)
     @Bean
     public SecurityFilterChain healthCheckSecurityFilterChain(HttpSecurity http) throws Exception {
-        return http.securityMatcher("/company-accounts-web/healthcheck").authorizeHttpRequests(
-                        auth -> auth.requestMatchers("/company-accounts-web/healthcheck").permitAll())
+        return ChsCsrfMitigationHttpSecurityBuilder.configureWebCsrfMitigations(
```

Comment:
Could be `configureApiCsrfMitigations` instead as it's only a GET endpoint, but doesn't do any harm